test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/ConcurrentHashMapTest.scala

```diff
@@ -194,6 +196,42 @@ class ConcurrentHashMapTest extends MapTest {
         assertTrue(s"missing value '$value'", encounteredValues.contains(value))
     }
   }
+
+  @Test def keySetWithMappedValue(): Unit = {
+    assumeTrue("Assumed JS or JDK>8", !executingInJVM || !executingInJVMOnJDK8OrLower)
```

Comment:
https://github.com/scala-js/scala-js/pull/4367/commits/21ff1ea6db82f7b27b0db10e247a9b13abe24d3b
Removed `assume`, since `keySet(null)` throwing NPE on JDK 8 on my end too.